src/main/java/com/echobox/api/linkedin/types/people/FollowerResult.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.echobox.api.linkedin.types.people;
+
+
+import com.echobox.api.linkedin.jsonmapper.LinkedIn;
+import lombok.Getter;
+import lombok.Setter;
+
+/**
+ * Represents a follower result returned by the People Typeahead API.
+ *
+ * @author Myrto Papakonstantinou
+ */
+
+public class FollowerResult {
```

Comment:
Minor and could be ignored, but you could put a @Getter and @Setter on the class instead of each individual property.